src/org/lsst/ts/jenkins/components/Csc.groovy

```diff
@@ -33,6 +33,46 @@ def build_conda(label) {
     """
 }
 
+def build_idl_conda(label) {
+    sh """
+        echo 'The XML version: ${params.xml_version}'
+        echo 'The SAL version: ${params.sal_version}'
+        echo 'The BuildType: ${params.build_type}'
+    """
+    if ( params.build_type == "Bleed" ) {
+        rpm_repo = "lsst-ts-bleed"
+    } else if ( params.build_type == "Daily" ) {
+        rpm_repo = "lsst-ts-daily"
+    } else if ( params.build_type == "Release" ) {
+        rpm_repo = "lsst-ts"
+    } else {
+        currentBuild.result = 'ABORTED'
+        error('Please properly define the build_type parameter.')
+    }
+    timeout(5) {
+        waitUntil(initialRecurrencePeriod: 15000, quiet: true) {
+            // The RPMs can take a few minutes to appear in the repo. This will wait 5 minutes then fail the build if the RPM is not found.
+            def r = sh (
+                script: "yum clean all ; " +
+                    " yum list --enablerepo=${rpm_repo} ts_sal_runtime-${params.xml_version}-${params.sal_version}.el7.x86_64 ",
+                returnStatus: true
+            )
+            return r == 0
+        }
+    }
+    sh """
+        # yum clean all ; yum makecache fast; yum update ; 
+        yum install -y --enablerepo=${rpm_repo} ts_sal_runtime-${params.xml_version}-${params.sal_version}.el7.x86_64
+        cd ${HOME}/conda
+        source /home/saluser/miniconda3/bin/activate 
+        conda config --add channels conda-forge
+        conda config --add channels lsstts
+        source ${OSPL_HOME}/release.com
+        conda build -c lsstts/label/${label} --prefix-length 100 .
+        ls /home/saluser/miniconda3/conda-bld/linux-64
```

Comment:
Do you need an `ls` command at the end of this function here.

Reply:
I removed the left-over debugging statement.